config/runtime.exs

```diff
@@ -195,3 +195,11 @@ case Keyword.get(args, :log_file) do
       timestamp_key: "ts",
       timestamp_format: :iso8601
 end
+
+# Sentry
+
+{git_sha, 0} = System.cmd("git", ["rev-parse", "--short", "HEAD"])
```

Comment:
You probably want to use this instead? https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

`GITHUB_SHA`?

Reply:
No. That would only work for GitHub workflows. We want this to work automatically on the server (i.e. without us manually fetching the commit's hash).

Reply:
ok, makes sense